changelogs/2.11.4.md

```diff
@@ -0,0 +1,161 @@
+# 2.11.4
+
+Date: 2024-08-16
+Tag: 2.11.4
+
+## Overview
+
+2.11.4 is the next stable release in the [long-term support (LTS)
+version][release_policy][release_policy] 2.11.x release series.
+
+The label "stable" means there are 2.11.x-based applications running in
+production for quite a while without known crashes, incorrect results or
+other showstopper bugs.
+
+This release resolves roughly 41 issues since the 2.11.3 version.
```

Comment:
Seems more like 51. The tool counts changelog entries and the LuaJIT section has 2 entries, but 12 bullets.

Reply:
Agree. Fixed.